tests/cli/test_init.py

```diff
@@ -79,7 +79,7 @@ def test_init(isolated_runner):
     assert 0 == result.exit_code
     assert new_project.exists()
     assert (new_project / '.renku').exists()
-    assert (new_project / '.renku' / 'metadata.yml').exists()
```

Comment:
shouldn't the metadata file still get created?

Reply:
Good point, it makes sense to still check `metadata` in the cli tests and change that only on template validation tests